cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile

```diff
@@ -67,86 +65,89 @@ add-site: ## add a new site to the site factory
 			--output-dir /app/sites \
 			--config-file /app/.cookiecutter_default.yml \
 			/app/template
+.PHONY: generate-site
 
+add-site: generate-site ## add a new site to the site factory
+	@echo "RICHIE_SITE=$(shell ls -td sites/* | head -1 | cut -f2 -d'/')" > .env
 .PHONY: add-site
 
-.site: ## site should be activated
+.env: ## site should be activated
 	@bin/activate
 
 # == Docker
-build: .site ## build all containers
+build: .env ## build all containers
 	$(COMPOSE) build app
 	$(COMPOSE) build nginx
 	$(COMPOSE) build app-dev
 .PHONY: build
 
-reset: .site ## Remove database and local files
+reset: .env ## Remove database and local files
 	$(COMPOSE) stop
 	rm -Ir data/* || exit 0
 	$(COMPOSE) rm db
 .PHONY: reset
 
-down: .site ## stop & remove containers
+down: .env ## stop & remove containers
 	@$(COMPOSE) down
 .PHONY: down
 
-logs: .site ## display app logs (follow mode)
+logs: .env ## display app logs (follow mode)
 	@$(COMPOSE) logs -f app-dev
 .PHONY: logs
 
-run: .site ## start the wsgi (production) or development server
+run: .env ## start the wsgi (production) or development server
 	@$(COMPOSE) up -V -d redis-sentinel
 	@$(WAIT_SENTINEL)
 	@$(COMPOSE) up -d nginx
 	@$(COMPOSE) up -d app-dev
 	@$(WAIT_DB)
 .PHONY: run
 
-stop: .site ## stop the development server
+stop: .env ## stop the development server
 	@$(COMPOSE) stop
 .PHONY: stop
 
-info: .site ## get activated site info
+info: .env ## get activated site info
 	@echo "RICHIE_SITE: $(COLOR_INFO)$(RICHIE_SITE)$(COLOR_RESET)"
 .PHONY: info
```

Comment:
All other commands seems to work but this one nope.
```
$> make info
RICHIE_SITE:
```

Reply:
Good catch!! Easy one :wink: